src/formatDate.js

```diff
@@ -50,7 +50,44 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let newDate = date;
+  const dateArray = newDate.split(fromFormat[3]);
+  const newDateArray = [];
+  let yearIndex = 0;
```

Comment:
this variable also seems redundant, it only adds unnecessary code where you use it